tests/composite_pk/tests.py

```diff
@@ -147,6 +147,21 @@ def test_in_bulk(self):
         result = Comment.objects.in_bulk([self.comment.pk])
         self.assertEqual(result, {self.comment.pk: self.comment})
 
+    @unittest.mock.patch.object(
+        type(connection.features), "max_query_params", new_callable=lambda: 10
+    )
```

Comment:
The OperationalError can be seen on SQLite by removing this mock. Without the mock, the test is quite slow due to ticket-36380.